src/styles/style.js

```diff
@@ -411,7 +411,10 @@ export var Style = {
 
     makeMesh (vertex_data, vertex_elements, options = {}) {
         let vertex_layout = this.vertexLayoutForMeshVariant(options.variant);
-        return new VBOMesh(this.gl, vertex_data, vertex_elements, vertex_layout, options);
+        return new VBOMesh(this.gl,
+            vertex_data, vertex_elements, vertex_layout,
+            { ...options, wireframe: debugSettings.wireframe }
```

Comment:
Moved wireframe option enable here, rather than having it hard-coded inside the `VBOMesh`, which (ideally) is more generic (though wireframe mode is itself hard-coded to specific data/draw mode format...).